changelog.d/1610.misc

```diff
@@ -0,0 +1 @@
+IrcBridge.memberListSyncers: Object -> Map.
```

Comment:
This needs to be much more descriptive :). Needs to identify the change from the end user perspective.